### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by z4cope**
- Added core structure files for Webpack.
- Generated the list items via Javascript.
- Added markup & styles using Webpack. 
- Configured Linters and made sure that there were no errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 - Webpack
+- Linters
 
-## Project Link on Github pages
+## Project Link on Github
 
-https://github.com/z4cope/webpack-boilerplate
+https://github.com/z4cope/Todolist-webapp
 
 ## Getting Started
```
Comment (mckent05) [inline README.md:17]: - To make your repo look more professional, I suggest you include steps on how to clone, navigate and open your project locally. Please fix this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @z4cope,

# STATUS: CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Your project works as expected 👍 👍 
- All linter checks are passing ✔️ 
- Your repo looks professional 👏 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (z4cope) [reply README.md:17]: @mckent05 I'm really thankful for the time you invested reviewing my project 😄, I thought you are reviewing only the pull request, as this mistake is on the main branch which when if I merge this branch It's going to be fixed 

- Regarding the steps for launching the project I've included them already as shown in the screenshot below, I think I may have just missed the npm start command which allows the project to run on the local host and I included it.
Screenshot 2022-11-02 at 4 58 59 PM

Comment (z4cope) [reply README.md:17]: @mckent05, Hello again, I think you are busy replying to my message, I'll just resubmit my project again.

**Review by ichala — APPROVED** [review]
Hi @z4cope  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

-Requirements Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**